packages/component/src/Localization/Localize.js

```diff
@@ -110,7 +110,7 @@ function normalizeLanguage(language) {
   } else if (language.startsWith('pl')) {
     return 'pl-PL';
   } else if (language.startsWith('pt')) {
-    if (language === 'pt-BR') {
+    if (language === 'pt-br') {
```

Comment:
could you explain why this is necessary?

Reply:
in the first line of this function, the language is setted to lower case, and this condidion was never true.

Reply:
Ah I understand now, thank you!